src/main/java/org/frankframework/frankdoc/model/ConfigChild.java

```diff
@@ -132,7 +132,7 @@ private static FrankAnnotation getIbisDoc(FrankMethod method) {
 			ConfigChild selected = bucket.get(0);
 			result.add(selected);
 			if(selected.isDeprecated()) {
-				log.warn("From duplicate config children, only a deprecated one is selected. In mode {}, {} will not have a config child for {}",
+				log.error("From duplicate config children, only a deprecated one is selected. In mode {}, {} will not have a config child for {}",
```

Comment:
Als de setter deprecated is moet hij niet in de STRICT modus komen (voor zowel attributen als elementen). 
Ik snap de melding nog niet goed, waarom zie jij (Martijn) dit als WARN/ERROR? Dit zou toch helemaal niet voor moeten komen? Geen enkel deprecated element/attribuut moet in de STRICT xsd.

Reply:
Als alle config children die een key (role name en type) delen deprecated zijn, dan is er inderdaad geen probleem. Er is een probleem als er bij het ontdubbelen niet-deprecated config children worden weggelaten. Dan worden config children weggelaten uit de strict xsd, alleen omdat er een conflictered config child is dat deprecated is. Ik heb de logging aangepast zodat er alleen een fout is bij dit probleemgeval.